src/TabCollectionView.js

```diff
@@ -8,8 +8,6 @@ define(function (require) {
 
         tagName: 'ul',
 
-        className: 'tab-item-view',
```

Comment:
Is this class available as an option? It's not in the readme. Also, the defaults shouldn't change in case anyone is using them in the wild.